client/src/nv_ingest_client/util/util.py

```diff
@@ -31,6 +31,16 @@
 logger = logging.getLogger(__name__)
 
 
+class ClientConfigSchema:
```

Comment:
Could you add unit tests for this schema.

Reply:
added unit test